api-tabular.md

```diff
@@ -79,14 +79,16 @@ curl -X GET "https://tabular-api.data.gouv.fr/api/resources/1c5075ec-7ce1-49cb-a
 ```
 
 Trois liens sont alors référencés pour chaque ressource :
-- `/profile` : qui retourne la description générale des lignes et colonnes de la donnée tabulaire
-- `/data` : qui retourne la donnée en tant que telle, paginée et avec des paramètres de filtres et de tris documentés plus hauts
-- `/swagger` qui retourne la documentation technique de la ressource, qui décrit notamment les possibilités de tris et de filtres sur la route `/data`
+- `/profile/` : qui retourne la description générale des lignes et colonnes de la donnée tabulaire (générée à partir [d'un autre package que nous maintenons](https://github.com/datagouv/csv-detective?tab=readme-ov-file#output))
+- `/data/` : qui retourne la donnée en tant que telle, paginée et avec des paramètres de filtres et de tris documentés plus bas
+- `/swagger/` : qui retourne la documentation technique de l'accès à la ressource, avec notamment les possibilités de tris et de filtres sur la route `/data/`
 
-**TODO** : s'assurer qu'on a bien enlevé les trailing `/` en prod d'ici là.
+**TODO** : s'assurer qu'on a bien enlevé les trailing `/` en prod d'ici là (ou qu'il y a des redirections ? en tout cas harmoniser ce doc)
```

Comment:
There is no redirect, but both are supported, cf: https://github.com/datagouv/hydra/pull/158